src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -112,34 +112,27 @@ class VisEditorVisualization extends Component {
         };
       });
     }
-  }
+  };
 
   hasShowPanelIntervalValue() {
     const type = get(this.props, 'model.type', '');
+    const interval = get(this.props, 'model.interval') || AUTO_INTERVAL;
+    const isAutoOrGteInterval = interval === AUTO_INTERVAL
```

Comment:
`isIntervalValid` already checks the `interval === AUTO_INTERVAL` condition.

Reply:
@Avinar-24  please review it more carefully. The main idea here that 'isAutoOrGteInterval' is true only for 'auto'/ 'gte' intervals or if ****isIntervalValid** is invalid**